src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private static final FigureName figureName = FigureName.SQUARE;
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nFigure:\t" + figureName
+                + ",\tColor:\t" + this.getColor()
+                + ",\tArea:\t" + this.getArea()
+                + ",\tSide:\t" + this.getSide());
```

Comment:
Use only English in messages/code. Instead of 'Figure:', use 'Shape:', to make the code more universal.